com.locomalito.abbayedesmorts.appdata.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
+  <id>com.locomalito.abbayedesmorts</id>
+  <metadata_license>CC-BY-3.0</metadata_license>
+  <project_license>GPLv3</project_license>
+  <content_rating type="oars-1.1" />
```

Comment:
Not sure if that's a good rating of the game.

Reply:
My fault, using this https://hughsie.github.io/oars/generate.html  I come up with:
Although I'm unsure if the character dying when touching a monster would be mild or moderate.

  <content_rating type="oars-1.0">
    <content_attribute id="violence-cartoon">moderate</content_attribute>
    <content_attribute id="violence-fantasy">moderate</content_attribute>
  </content_rating>

Reply:
fixed in be7b9b3be258120a6750cb58061a09da1acf6c2f